CryptKeeperZK/crypt-keeper-extension
# fix: delete identity

## Explanation

There is a bug with `IdentityService` because it use two separate stores to keep identities.
In this PR I removed inheritance from `SimpleStorage` for `IdentityStorage` and use composition instead.
In addition, I added some tests for `IdentityList` component and refactored it.

Details are below:
- [x] Simplify identity service and do not use inherited storage
- [x] Improve types
- [x] Move IdentityList component from Home component file and add tests
- [x] Refactoring for IdentityList component

## More Information

Fixes #133

## Screenshots/Screencaps

N/A

## Manual Testing Steps

1. Try to add new identity
2. Remove this identity
3. Reload page or lock extension
4. You will see identity list without this identity

## Pre-Merge Checklist


### + If there are functional changes:


> PR template source from [github.com/MetaMask](https://github.com/MetaMask)
